pkg/topo/get.go

```diff
@@ -71,7 +76,44 @@ func getGetKindCommand() *cobra.Command {
 }
 
 func runGetEntityCommand(cmd *cobra.Command, args []string) error {
-	return runGetCommand(cmd, args, topoapi.Object_ENTITY)
+	// if any flag relating to the entity-relation filter is set, call the corresponding function (which checks if all necessary flags are set)
+	if cmd.Flag("related-to").Value.String() != "" || cmd.Flag("related-via").Value.String() != "" || cmd.Flag("tgt-kind").Value.String() != "" {
```

Comment:
use "cmd.Flags().GetString("...") to get a string... also I'd use len() > 0 test.